tico/passes/remove_redundant_reshape.py

```diff
@@ -427,3 +428,113 @@ def call(self, exported_program: ExportedProgram) -> PassResult:
         graph_module.recompile()
 
         return PassResult(modified)
+
+
+@trace_graph_diff_on_pass
+class RemoveRedundantReshapePattern6(PassBase):
+    def __init__(self):
+        super().__init__()
+        self.SHAPE_PRESERVING_OPS = [
+            *ops.aten.mul_scalar,
+            *ops.aten.mul_tensor,
+            *ops.aten.add,
+            # TODO add more ops.
+        ]
```

Comment:
They are not shape-preserving Ops. The output shape can change due to broadcasting.